From bed646918bc67e9e2151ccbda03aae34717821fe Mon Sep 17 00:00:00 2001
From: Andrei Gherzan <andrei.gherzan@huawei.com>
Date: Wed, 20 Apr 2022 14:57:14 +0200
Subject: [PATCH] Drop ar 'u' argument

binutils/ar is configured in deterministic mode by default making the
'u' argument irrelevant while leading to warning massages similar to:

| ar: `u' modifier ignored since `D' is the default (see `U')

We also add 'D' flag explicitely to not rely in the default configuration
for reproducible archives.

Upstream-Status: Inappropriate [OE specific]
Signed-off-by: Andrei Gherzan <andrei.gherzan@huawei.com>
---
 bin/automake.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/automake.in b/bin/automake.in
index 92bcebe..0f2b84b 100644
--- a/bin/automake.in
+++ b/bin/automake.in
@@ -2678,7 +2678,7 @@ sub handle_libraries ()
     }
 
   define_variable ('AR', 'ar', INTERNAL);
-  define_variable ('ARFLAGS', 'cru', INTERNAL);
+  define_variable ('ARFLAGS', 'crD', INTERNAL);
   define_verbose_tagvar ('AR');
 
   foreach my $pair (@liblist)
-- 
2.25.1

